dvc/state.py

```diff
@@ -370,13 +370,12 @@ def save(self, path_info, checksum):
         assert path_info.scheme == "local"
         assert checksum is not None
 
-        path = fspath_py35(path_info)
-        assert os.path.exists(path)
+        assert os.path.exists(path_info)
```

Comment:
this one actually has to be 
```suggestion
        assert os.path.exists(fspath_py35(path_info))
```
because os.path.exists started supporting path-objects after 3.5